tendrl-monitoring-integration.spec

```diff
@@ -41,9 +43,9 @@ install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/monitoring-integ
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration/grafana
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration/grafana/dashboards
-#install -Dm 0644 monitoring-integration.service $RPM_BUILD_ROOT%{_unitdir}/monitoring-integration.service
-install -Dm 0644 etc/grafana/grafana.ini $RPM_BUILD_ROOT%{_sysconfdir}/grafana/grafana.ini
 install -Dm 0644 etc/tendrl/monitoring-integration/monitoring-integration.conf.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration/monitoring-integration.conf.yaml
+install -Dm 0644 etc/grafana/grafana.ini $RPM_BUILD_ROOT%{_sysconfdir}/grafana/grafana.ini
+install -Dm 0644 tendrl-monitoring-integration.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-monitoring-integration.service
```

Comment:
@TimothyAsir I converted  monitoring-integration to a service, but when I run the rpm build command to generate rpm it throws an error saying that its not able to locate tendrl-monitoring-integration.service.
The work around is I copy tendrl-monitoring-integration to etc/. and then change path in .spec file and it works